app/components/collections/header_component.html.erb

```diff
@@ -0,0 +1,23 @@
+<header class="title">
```

Comment:
extracted from `views/collections/show.html.erb` since it's now used by Collections#show and Works#index